package.json

```diff
@@ -15,8 +15,9 @@
     "gatsby-plugin"
   ],
   "scripts": {
-    "build": "babel src --out-dir . --ignore __tests__",
-    "watch": "babel -w src --out-dir . --ignore __tests__"
+    "build": "babel src --out-dir dist --ignore __tests__",
+    "prepare": "yarn build",
```

Comment:
When running `npm publish`, it will first run the `prepare` script automatically if it's there.

In this case, it will `build` the code and put it into the `/dist` folder...which we want. Then the developer does not have to remember to build it before they publish.